source/configure-components/index.html.md.erb

```diff
@@ -23,44 +27,18 @@ Read the [JavaScript API Reference](../javascript-api-reference/) for the list o
 
 The examples below follow our recommended [Import JavaScript using a bundler](../import-javascript/#import-javascript-using-a-bundler) approach and demonstrate how to:
 
-- [configure individual component instances](#configure-individual-component-instances)
-- [configure all components using the initAll function](#configure-all-components-using-the-initall-function)
+- [configure instances of specific components using the `createAll` function](#configure-instances-of-specific-components-using-the-createall-function)
+- [configure all components using the `initAll` function](#configure-all-components-using-the-initall-function)
 - [check for JavaScript errors in the browser console](#javascript-errors-in-the-browser-console)
 
-### Configure individual component instances
-
-Component constructors accept two arguments:
-
-1. The HTML element that represents the component.
-2. An optional configuration object.
-
-Although JavaScript configuration objects are optional, configuration can still be provided or overridden by [HTML data attributes](#adding-html-data-attributes).
-
-To learn more about how configuration is passed from Nunjucks macros to HTML data attributes, see advanced examples in [the localisation options](../localise-govuk-frontend/).
-
-#### Without configuration object
-
-You can select and initialise a component using its `data-module` attribute. For example, use `govuk-character-count` to initialise the first character count component on a page:
-
-```javascript
-import { CharacterCount } from 'govuk-frontend'
-
-const $element = document.querySelector('[data-module="govuk-character-count"]')
-new CharacterCount($element)
-```
-
-When initialised individually, errors are thrown rather than logged. You must check your application works without errors or some components will not work correctly.
+### Configure instances of specific components using the `createAll` function
```

Comment:
@36degrees Updated the headings so they both match. @claireashworth does that make the heading too long?